.ruby-gemset

```diff
@@ -0,0 +1 @@
+sidekiq-monitoring
```

Comment:
This is created by RVM I guess? I don't think it should be in the gem repository

Reply:
If you want I can remove it, but I don't think it hurts to force a gemset if rvm is installed. I actually do this with all my projects (even if all of them are private :D) to avoid conflicts between them.

Reply:
I'm not using RVM but I see it's not recommended for a gem so I would say please remove it: https://stackoverflow.com/questions/16087120/should-i-check-in-ruby-gemset-and-or-ruby-version

Reply:
Done.